### ISSUE TITLE
TypeScript - QnA Maker Deployment Fails

### ISSUE THREAD

**Issue by thegoo** [microsoft/botframework-solutions#1578]
## Project

**Name**: Virtual Assistant

**Language**: TypeScript

## Description
"The account type QnA Maker is either invalid or unavailable in given region." Specifying **eastus** as the region for resources.

QnA Maker is available in westus.  Looked at the C# deployment ARM template with @lauren-mills and it is set to westus.

## To Reproduce

1. Create a new Virtual Assistant project in Visual Studio
2. Clean/build the solution
3. Launch PowerShell and navigate to the project folder
4. Execute the .\Deployment\Scripts\deploy.ps1 command
5. Provide a bot name, **specify _eastus_ for resources**, specify westus for LUIS, and provide authoring key
6. Observe the error in PowerShell or via the Azure Portal event log

## Expected behavior
Deployment to succeed.

## Screenshots
![image](https://user-images.githubusercontent.com/7401156/59610940-4e438f00-9112-11e9-858b-56da76e54cd6.png)

**Comment by darrenj** [comment]
Perhaps you have an old generator or a project created 2/3 weeks ago? You can validate to check template.json in your project has this fragment with qnamaker fixed to westus?

```
 {
      "comments": "Cognitive service key for all QnA Maker knowledgebases.",
      "type": "Microsoft.CognitiveServices/accounts",
      "kind": "QnAMaker",
      "apiVersion": "2017-04-18",
      "name": "[parameters('qnaMakerServiceName')]",
      "location": "westus",
      "sku": {
        "name": "[parameters('qnaMakerServiceSku')]"
      },
      "properties": {
        "apiProperties": {
          "qnaRuntimeEndpoint": "[concat('https://',reference(resourceId('Microsoft.Web/sites', parameters('qnaMakerWebAppName'))).hostNames[0])]"
        }
      },
```

**Comment by thegoo** [comment]
hey @darrenj - I pulled this down last week.  Looks like the deployment template was updated a day or two after I pulled and can confirm it refers to westus in the [source](https://github.com/microsoft/botframework-solutions/blob/master/templates/Virtual-Assistant-Template/typescript/generator-botbuilder-assistant/generators/app/templates/sample-assistant/deployment/resources/template.json) -- assuming I am in the right spot.

Has this been incorporated and published to npm?  I did a npm install generator-botbuilder-assistant and did yo botbuilder-assistant and the template.json still refers to: "location": "[parameters('qnaServiceLocation')]".

To double check, I uninstalled via npm, did an install, and did another generation with the same result.

//cc: @lauren-mills
